stm32-modules/include/flex-stacker/flex-stacker/ui_task.hpp

```diff
@@ -40,34 +44,67 @@ static auto color_to_channels(StatusBarColor color) -> const ChannelMapping& {
             return green_channels;
         case StatusBarColor::Blue:
             return blue_channels;
+        case StatusBarColor::Yellow:
+            return yellow_channels;
         default:
             return white_channels;
     }
 }
 
 // The timer driving LED update frequency should run at this period
-static constexpr uint32_t UPDATE_PERIOD_MS = 1000;
+static constexpr uint32_t UPDATE_PERIOD_MS = 1;
```

Comment:
that seems like quite a lot, it's the same frequency as the systick. can we make it 10 or something?